tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/deid/fitbit_dateshift_test.py

```diff
@@ -175,6 +187,25 @@ def test_dateshifting(self, mock_schema):
                 (802, shifted_date + four_days, shifted_datetime + four_days,
                  shifted_timestamp + four_days, 'jane doe')
             ]
+        }, {
+            'name':
+                self.fq_table_names[1].split('.')[-1],
+            'fq_table_name':
+                self.fq_table_names[1],
+            'fq_sandbox_table_name':
+                '',
+            'fields': [
+                'person_id', 'p_date', 'p_datetime', 'p_timestamp', 'p_name'
+            ],
+            'loaded_ids': [803, 804],
+            'sandboxed_ids': [],
+            'cleaned_values': [
+                (803, shifted_date + four_days, shifted_datetime + four_days,
```

Comment:
The cleaned values here are based on the offset from the shifted date/shifted_datetime of `2016-05-01`.  I had to trace the records and the date shifts to confirm, and it looks correct.